src/chapters/style/documentation.typ

```diff
@@ -0,0 +1,247 @@
+#import "/src/util.typ": *
+#import mantys: *
+
+= Documentation <sec:style:docs>
+#wbox[
+  This section documents the syntax of documentation commens, see @sec:api:docs for the contents and purpose of documentation comments.
+]
+
+Documentation is placed on special comments using three forward slashes `///` and an optional space.
```

Comment:
> and an optional space.

Some people from other regions may usually use other kind of whitespace. We can emphasize it, but it is a personal preference:

```patch
- and a space.
+ and an ascii space (ASCII 32 or `U+0020`).
```